src/main/java/com/conveyal/r5/analyst/TemporalDensityResult.java

```diff
@@ -0,0 +1,95 @@
+package com.conveyal.r5.analyst;
+
+import com.conveyal.r5.analyst.cluster.AnalysisWorkerTask;
+import com.google.common.base.Preconditions;
+
+import static com.conveyal.r5.common.Util.notNullOrEmpty;
+import static com.conveyal.r5.profile.FastRaptorWorker.UNREACHED;
+
+/**
+ * An instance of this is included in a OneOriginResult for reporting how many opportunities are encountered during each
+ * minute of travel. If we use more than one destination point set they are already constrained to all be aligned with
+ * the same number of destinations.
+ *
+ * The data retained here feed into three different kinds of results: "Dual" accessibility (the number of opportunities
+ * reached in a given number of minutes of travel time); temporal opportunity density (analogous to a probability density
+ * function, how many opportunities are encountered during each minute of travel, whose integral is the cumulative
+ * accessibility curve).
```

Comment:
Perhaps part of this comment was unintentionally deleted? Otherwise,

```suggestion
 * The data retained here feed into two different kinds of results: "Dual" accessibility (the number of minutes of
 * travel time needed to reach a given number of opportunities); and temporal opportunity density (analogous to a probability density
 * function, how many opportunities are encountered during each minute of travel, whose integral is the cumulative
 * accessibility curve).
```